src/robotide/editor/texteditor.py

```diff
@@ -85,6 +85,7 @@ def f(event):
         self.register_shortcut('CtrlCmd-Z', focused(lambda e: self._editor.undo()))
         self.register_shortcut('CtrlCmd-Y', focused(lambda e: self._editor.redo()))
         # self.register_shortcut('Del', focused(lambda e: self._editor.delete()))
+        self.register_shortcut('CtrlCmd-D', focused(lambda e: self._editor.delete_row(e)))
```

Comment:
Using this registering does not overrides the Ctrl-D from Scintilla library, and the command is "duplicate text". My PR has this line commented.